theme/templates/accounts/profile.html

```diff
@@ -4,78 +4,521 @@
+                            </tr>
+                        {% endif %}
 
-            {% if profile.subject_areas %}
-            <li><strong>Subject areas:</strong> {{ profile.subject_areas }}</li>
-            {% endif %}
+                        {% if profile.profession %}
+                            <tr>
+                                <td><span class="text-muted">Profession</span></td>
+                                <td>{{ profile.profession }}</td>
+                            </tr>
+                        {% endif %}
+                        </tbody>
+                    </table>
+                    {% endif %}
+                </div>
+            </div>
+        </div>
 
-            {% if profile.organization %}
-            <li><strong>Organization:</strong> {{ profile.organization }}</li>
-            {% endif %}
+        <div class="full-width-tabs-container">
+            <!-- Nav tabs -->
+            <ul class="nav nav-tabs full-width-tabs" role="tablist">
+                <li role="presentation" class="active">
+                    <a href="#overview" aria-controls="home" role="tab"
+                       data-toggle="tab"><i class="glyphicon glyphicon-list-alt"></i> OVERVIEW</a></li>
+                <li role="presentation">
+                    <a href="#contributions" aria-controls="profile" role="tab" data-toggle="tab">
+                        <i class="glyphicon glyphicon-file"></i> CONTRIBUTIONS</a>
+                </li>
+{#                <li role="presentation">#}
+{#                    <a href="#social" aria-controls="messages" role="tab"#}
+{#                       data-toggle="tab"><i class="glyphicon glyphicon-user"></i> SOCIAL</a></li>#}
+            </ul>
 
-            {% if profile.organization_type %}
-            <li><strong>Organization Type:</strong> {{ profile.organization_type }}</li>
-            {% endif %}
+            <!-- Tab panes -->
+            <div class="tab-content">
+                <div role="tabpanel" class="tab-pane fade in active" id="overview">
+                    <div class="row">
+                        <div class="col-md-7">
+                            <div class="panel panel-default">
+                                <div class="panel-heading">
+                                    <h3 class="panel-title">Recent activity</h3>
+                                </div>
+                                <div class="panel-body">
+                                    {% if resources %}
 
-            {% if profile.phone_1 %}
-            <li><strong>Phone 1:</strong> {{ profile.phone_1 }} {{ profile.phone_1_type }}</li>
-            {% endif %}
+                                    {% for res in resources reversed %}
+                                        {# Show only the last five #}
+                                        {% if forloop.counter0 < 5 %}
+                                        <div class="activity-block">
```

Comment:
I'd suggest to merge this work now and fix it in another issue. @mauriel133 Could you create an issue for having the resource sorted in the back end to return the sorted resource sub-list for front-end use? @ironfroggy what do you think? Could you give a quick scan-through on this PR to see whether you can give a +1 to have it merged tomorrow morning so I can get started to deploy for release 1.6.2?